crates/ruff/src/rules/flake8_comprehensions/fixes.rs

```diff
@@ -25,65 +25,43 @@ use crate::{
 };
 
 /// (C400) Convert `list(x for x in y)` to `[x for x in y]`.
-pub(crate) fn fix_unnecessary_generator_list(
-    locator: &Locator,
-    stylist: &Stylist,
-    expr: &Expr,
-) -> Result<Edit> {
-    // Expr(Call(GeneratorExp)))) -> Expr(ListComp)))
-    let module_text = locator.slice(expr.range());
-    let mut tree = match_expression(module_text)?;
-    let call = match_call_mut(&mut tree)?;
-    let arg = match_arg(call)?;
+pub(crate) fn fix_unnecessary_generator_list(expr: &Expr, locator: &Locator) -> Result<Edit> {
```

Comment:
Did you refactor this code? Should it be part of this PR?

Reply:
Sorry, I think this was accidentally included when I was playing around with removing LibCST. I must've gotten my branches confused.